imap_processing/tests/spice/test_pointing_frame_handler.py

```diff
@@ -0,0 +1,182 @@
+"""Tests Pointing Frame Generation."""
+
+import shutil
+import sys
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+import spiceypy as spice
+
+from imap_processing.spice.pointing_frame_handler import (
+    average_quaternions,
+    create_pointing_frame,
+    create_rotation_matrix,
+    get_et_times,
+)
+
+
+@pytest.fixture()
+def kernel_path(tmp_path):
+    """Create path to kernels."""
+
+    test_dir = (
+        Path(sys.modules[__name__.split(".")[0]].__file__).parent
+        / "tests"
+        / "spice"
+        / "test_data"
+    )
+
+    kernels = [
+        "naif0012.tls",
+        "imap_science_0001.tf",
+        "imap_sclk_0000.tsc",
+        "imap_wkcp.tf",
+        "imap_spin.bc",
+    ]
+
+    for file in test_dir.iterdir():
+        if file.name in kernels:
+            shutil.copy(file, tmp_path / file.name)
+
+    return tmp_path
+
+
+@pytest.fixture()
+def kernels(kernel_path):
+    """Create kernel list."""
+    kernels = [str(file) for file in kernel_path.iterdir()]
+
+    return kernels
+
+
+@pytest.fixture()
+def ck_kernel(kernel_path):
+    """Create ck kernel."""
+    ck_kernel = [
+        str(file) for file in kernel_path.iterdir() if file.name == "imap_spin.bc"
+    ]
+
+    return ck_kernel
```

Comment:
There is a fixture that defines the spice test data path. This function would be clearer if you did:
```suggestion
def ck_kernel(spice_test_data_path):
    """Location of ck kernel to create pointing kernel from."""
    return spice_test_data_path / "imap_spin.bc"
```